frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -171,6 +164,20 @@ action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=Slide
 action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected branch
 
 
+# Graph Table
+
+action.GitMachete.DiscoverAction.text=Discover Branch Layout
+action.GitMachete.DiscoverAction.description=Discovers the Branch Tree in Git Machete Plugin
```

Comment:
I think all descriptions are infinitive, not 3rd person (at least they should be AFAIK, pls verify while you're here)

Reply:
I can't find any information about it on JetBrains pages.
Ok, changed to infinitive